tests/InputFiles/CurvedScalarWave/WorldtubeKerrSchild.yaml

```diff
@@ -26,6 +26,12 @@ ResourceInfo:
     WorldtubeSingleton:
       Proc: Auto
       Exclusive: false
+    Axis1:
+      Proc: Auto
```

Comment:
If all of these are "Auto" and "false, you can just set the overall singleton block to "Singletons: Auto" to simplify things